src/webgpu/shader/validation/parse/file.spec.ts

```diff
@@ -0,0 +1,30 @@
+export const description = `Validation tests for file parsing`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+g.test('valid_file')
```

Comment:
It's a little strange that this is called "valid_file" when the thing being tested isn't a file, but a source string. Maybe "valid_source", "source.spec.ts", etc?

Reply:
done